session/session.go

```diff
@@ -575,6 +578,9 @@ func (s *session) commitTxnWithTemporaryData(ctx context.Context, txn kv.Transac
 		if tbl.GetMeta().TempTableType != model.TempTableLocal {
 			continue
 		}
+		if _, ok := localTempTables.TableByID(tblID); !ok && tbl.GetMeta().TempTableType == model.TempTableLocal {
```

Comment:
`tbl.GetMeta().TempTableType == model.TempTableLocal`  is unnecessary because it is always true here.